content/project/governance.adoc

```diff
@@ -90,7 +92,13 @@ We encourage plugins to follow the same 3rd party license policy as the core. Yo
 
 == Trademark
 
-The name "Jenkins" is a registered trademark in the USA (link:https://trademarks.justia.com/854/47/jenkins-85447465.html[#4664929], held by SPI) in order to protect the project and users from confusing use of the term. To reduce the process overhead and uphold our open-source spirit, we adopt the Linux kernel policy on trademark usage.
+The name "Jenkins" is a registered trademark in the USA
+(link:https://trademarks.justia.com/854/47/jenkins-85447465.html[#4664929],
```

Comment:
Terrible line breaks as usual (and GitHub considers all headers in the document changed in the formatted diff) :stuck_out_tongue: